test/fuzz/basic.ml

```diff
@@ -477,21 +477,22 @@ let expr_call (env : Env.t) (stack : val_type list) =
       | _ -> None )
     env.funcs
 
-let expr_br_if (env : Env.t) (stack : val_type list) =
+let expr_br_if (env : Env.t) (stack : val_type list) (is_block : bool) =
```

Comment:
Could you define a something like:

```ocaml
type label_kind =
| Loop
| Block
| Func (* I'm not sure this will be needed *)
```

and use this later in the code rather than the boolean ?